Content.Server/PDA/PDASystem.cs

```diff
@@ -188,14 +260,63 @@ private void AfterUIOpen(EntityUid uid, PDAComponent pda, AfterActivatableUIOpen
             if (!_uiSystem.TryGetUi(pda.Owner, PDAUiKey.Key, out var ui))
                 return;
 
-            var ownerInfo = new PDAIdInfoText
+            var ownerInfo = new PdaIdInfoText
             {
                 ActualOwnerName = pda.OwnerName,
                 IdOwner = pda.ContainedID?.FullName,
                 JobTitle = pda.ContainedID?.JobTitle
             };
 
-            var state = new PDAUpdateState(pda.FlashlightOn, pda.PenSlot.HasItem, ownerInfo, pda.StationName, true, HasComp<InstrumentComponent>(pda.Owner), GetDeviceNetAddress(pda.Owner));
+            var currentTime = GameTicker.RoundDuration();
+            var stationTime = new StationTimeText
+            {
+                Hours = currentTime.Hours.ToString(),
+                Minutes = currentTime.Minutes.ToString(),
+            };
```

Comment:
This will never update while the UI is open, it should just send a timespan of roundstart time probably and then client just handles it itself (also subtracting any paused time for the entity).